actors/builtin/miner/miner_actor.go

```diff
@@ -131,7 +131,8 @@ func (a Actor) ChangeWorkerAddress(rt Runtime, params *ChangeWorkerAddressParams
 	})
```

Comment:
ControlAddresses method needs a call to `rt.ValidateImmediateCallerAcceptAny()` on line 103 to satisfy the caller verification invariant required by spec.  Determined while using this branch to integrate go-filecoin.

Reply:
orthogonal so put it up here: https://github.com/filecoin-project/specs-actors/pull/202

Reply:
thats.... dumb